content/cluster-installation/vmware/ipi-proxy.md

```diff
@@ -124,10 +137,138 @@ sshKey: "....."                                   <===== Added
 pullSecret: '{"auths":{"cloud.openshift.com":....'
 ```
 
-Here all details about install-config.yaml
+### Example install-config.yaml Static IPs
 
+```yaml
+additionalTrustBundlePolicy: Proxyonly
+apiVersion: v1
+baseDomain: openshift.jarvis.lab
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  platform: {}
+  replicas: 3
+metadata:
+  name: ocp-mk1
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.10.80.0/24   #same L2 as k8s-machines
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  vsphere:
+    hosts:
+    - role: bootstrap
+      networkDevice:
+        ipAddrs:
+        - 10.10.80.99/24
+        gateway: 10.10.80.1
+        nameservers:
+        - 10.10.10.53
+    - role: control-plane
+      networkDevice:
+        ipAddrs:
+        - 10.10.80.151/24
+        gateway: 10.10.80.1
+        nameservers:
+        - 10.10.10.53
+    - role: control-plane
+      networkDevice:
+        ipAddrs:
+        - 10.10.80.152/24
+        gateway: 10.10.80.1
+        nameservers:
+        - 10.10.10.53
+    - role: control-plane
+      networkDevice:
+        ipAddrs:
+        - 10.10.80.153/24
+        gateway: 10.10.80.1
+        nameservers:
+        - 10.10.10.53
+    - role: compute
+      networkDevice:
+        ipAddrs:
+        - 10.10.80.154/24
+        gateway: 10.10.80.1
+        nameservers:
+        - 10.10.10.53
+    - role: compute
+      networkDevice:
+        ipAddrs:
+        - 10.10.80.155/24
+        gateway: 10.10.80.1
+        nameservers:
+        - 10.10.10.53
+    - role: compute
+      networkDevice:
+        ipAddrs:
+        - 10.10.80.156/24
+        gateway: 10.10.80.1
+        nameservers:
+        - 10.10.10.53
+    apiVIPs:
+    - 10.10.80.100
+    failureDomains:
+    - name: failure-domain
+      region: homelab-region
+      server: vcsa.jarvis.lab
+      topology:
+        computeCluster: /dc-1/host/cl-1
+        datacenter: dc-1
+        datastore: /dc-1/datastore/vsanDatastore
+        networks:
+        - 1080-wkld-net
+        resourcePool: /dc-1/host/cl-1/Resources/ocp-mk1
+        folder: /dc-1/vm/ocp-mk1
+      zone: zone1
+    ingressVIPs:
+    - 10.10.80.102
+    vcenters:
+    - datacenters:
+      - dc-1
+      password: ******
+      port: 443
+      server: vcsa.jarvis.lab
+      user: administrator@vsphere.local
+publish: External
+proxy:
+  httpProxy: http://172.16.0.1:3128/
+  httpsProxy: http://172.16.0.1:3128/
+  noProxy: 172.16.0.0/24,apps.vde.openshift.pub
```

Comment:
Please replace to your cluster name and base domain.

Reply:
Good point! Machine CIDR and vCenter IP are also a must according to our docs: https://docs.openshift.com/container-platform/4.17/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.html

Reply:
MachineCIDS should be added automatically.

Reply:
@rbo will this hinder you from merging the PR?

Reply:
nope :-)